packages/SketchMorph2-Core.package/M2TemporaryTextMorph.class/instance/dismissCurrent.st

```diff
@@ -0,0 +1,7 @@
+interaction
+dismissCurrent
+	self canvas ifNotNil: [self canvas removeMorph: self].
+	ActiveHand newKeyboardFocus: nil.
```

Comment:
This one is being set twice. Once in the sender of `dismissCurrent` and once here. I suggest removing this line.

Reply:
I'd suggest removing it in the sender since we are also setting the focus in `M2TemporaryTextMorph>>getAt:`, so I think the same class should be responsible for releasing the focus.